secure_fw/CMakeLists.txt

```diff
@@ -47,10 +47,18 @@ target_compile_options(tfm_s
         ${COMPILER_CP_FLAG}
 )
 
+if(${CMAKE_BUILD_TYPE} STREQUAL "Debug")
```

Comment:
We want to apply this workaround for all build types.

I want to be able to debug non-Debug builds as well.

e.g. Debug does all kinds of things and may increase the binary size such that I can't use Debug to debug my app. In which case I wish to debug the non-Debug build. And I want debug symbols for this.